### DISCUSSION TITLE
Folders for Team Queries

### DISCUSSION THREAD

**Discussion by Benny-Nottonson** [supabase/supabase#30536] [Feature Requests]
Hello! I have found it easy to store some of my frequently modified function definitions in SQL, and I share these with my team. Folders has been very useful for me to organize these, but I am unable to use these folders in the shared team folder, I think this would be a net benefit.

**Comment by opxdelwin** [comment]
+10 Bump for traction.

**Comment by memke1** [comment]
Looking for this feature as well :)

**Comment by John-Crawshaw** [comment]
Bump

**Comment by emersonp** [comment]
Surprised this didn't exist.
